lib/workers/repository/update/pr/changelog/release-notes.spec.ts

```diff
@@ -94,9 +95,16 @@ describe('workers/repository/update/pr/changelog/release-notes', () => {
   describe('addReleaseNotes()', () => {
     it('returns input if invalid', async () => {
       const input = { a: 1 };
-      expect(await addReleaseNotes(input as never)).toEqual(input);
-      expect(await addReleaseNotes(null)).toBeNull();
-      expect(await addReleaseNotes({ versions: [] } as never)).toStrictEqual({
+      expect(
+        await addReleaseNotes(input as never, {} as BranchUpgradeConfig)
```

Comment:
i think those `input` objects should have a proper  type, you can use `partial<TYPE>({ a: 1})` helper function from test utils

Reply:
This test is `returns input if invalid`, so I can't cast `{ a: 1}` to a `ChangeLogResult` object even partially. Actually, I didn't create this test case but it's changed due to the linting after adding a new parameter to the function